docs: new SnapX logo/screenshot & badges for packages

ok-coder1 opened the pull request.

ok-coder1 committed:
docs: add new and better badges with Homebrew, AUR and COPR

Signed-off-by: okcoder1 <codershub.code@gmail.com>

BrycensRanch commented:
This is an interesting PR 🤔

ok-coder1 committed:
docs: add COPR badge

Signed-off-by: okcoder1 <codershub.code@gmail.com>

ok-coder1 commented:
Would this be better? (me)
<a href="https://discord.gg/ys3ZCzttVQ"><img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/social/discord-singular_vector.svg" height="40" alt="Discord"/></a>
or (us)
<a href="https://discord.gg/ys3ZCzttVQ"><img src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/social/discord-plural_vector.svg" height="40" alt="Discord"/></a>

ok-coder1 commented:
And we probably need to add a copyright disclaimer that the Fedora logo is solely owned by the Fedora Project.

![COPR](https://raw.githubusercontent.com/ok-coder1/devins-badges-contrib/refs/heads/v3/assets/compact/available/copr_46h.png)

ok-coder1 commented:
Maybe we can use ![](https://docs.fedoraproject.org/en-US/project/_images/fonts.png) as recommended by https://docs.fedoraproject.org/en-US/project/brand/ for the font of the COPR badge.

ok-coder1 commented:
> This is an interesting PR 🤔

How?

BrycensRanch commented:
> > This is an interesting PR 🤔
> 
> How?

Because I'm about to hijack it for a README makeover

ok-coder1 commented:
> > > This is an interesting PR 🤔
> > 
> > 
> > How?
> 
> Because I'm about to hijack it for a README makeover

NOOOOOOO.
Well then, you make it better and I'll add this later I guess.

BrycensRanch commented:
> > > > This is an interesting PR 🤔
> > > 
> > > 
> > > How?
> > 
> > 
> > Because I'm about to hijack it for a README makeover
> 
> NOOOOOOO. Well then, you make it better and I'll add this later I guess.

Don't worry, my changes won't be destructive. Infact, they're going on this branch ;)

ok-coder1 committed:
style: sort alphabetically

Signed-off-by: okcoder1 <codershub.code@gmail.com>

ok-coder1 committed:
chore: use raw github links for badges

Signed-off-by: okcoder1 <codershub.code@gmail.com>

BrycensRanch committed:
chore: update application logo

Replaced the app logo with a new design. Notably, the logo’s colors
and circular shape look somewhat similar to Chrome’s, but this
similarity is unintended and purely coincidental.

This logo is subject to change this weekend. I need to shut off my computer soon to clean my fans. My computer sounds like a jet engine.

BrycensRanch committed:
chore: delete Logo sources folder

BrycensRanch committed:
docs: remove other now redudant other platforms section

BrycensRanch committed:
docs: update SnapX screenshot to use simple cursor

BrycensRanch committed:
chore: update SnapX_Logo.png resource

ok-coder1 committed:
chore: make the url point correctly

Signed-off-by: okcoder1 <codershub.code@gmail.com>

ok-coder1 committed:
style: sort alphabetically

Signed-off-by: okcoder1 <codershub.code@gmail.com>
